parsers/CL_SIC.py

```diff
@@ -276,11 +308,13 @@ def thermal_processer(df, logger):
     gas_generation = []
     oil_generation = []
     biomass_generation = []
+    geothermal_generation = []
     unknown_generation = []
 
-    for plant in thermal_plants.keys():
+    for plant in data_plants:
         plant_vals = thermal_df.loc[plant].to_dict()
-        plant_type = thermal_plants[plant]
+
+        plant_type = THERMAL_PLANTS.get(plant, 'unknown')
```

Comment:
sorry to be annoying, but should we log the case where a `plant_type` is not the `THERMAL_PLANTS` in order to add it in the future?

Reply:
L305 should catch that.

Reply:
ha! didn't see it. 👍